docs/features/security/auto-invalidate-secrets.md

```diff
@@ -1,57 +0,0 @@
----
```

Comment:
Let's bring this back please? Because this is something that is yet-to-be-released?

Reply:
Not sure what you mean but I didn't delete it. 
https://github.com/arcus-azure/arcus.backgroundjobs/pull/73/files/ff3ff8a48f3af3c827c23f452efae7a94e69b4a2#diff-21f5d2a3501524fd0d09614e93e314698c812fc712ccbfed80fbc9266aff5a93
To view it live:
https://deploy-preview-73--arcus-background-jobs-staging.netlify.app/